app/notification_email.go

```diff
@@ -283,6 +287,38 @@ func getFormattedPostTime(user *model.User, post *model.Post, useMilitaryTime bo
 	}
 }
 
+func (a *App) generateHyperlinkForChannels(postMessage, teamName, teamURL string) string {
+
+	team, err := a.GetTeamByName(teamName)
+	if err != nil {
+		mlog.Error(fmt.Sprintf("Encountered error while looking up team by name for %s with error: %s", teamName, err.Error()))
+		return postMessage
+	}
+
+	channelNames := model.ChannelMentions(postMessage)
+	if len(channelNames) == 0 {
+		mlog.Debug(fmt.Sprintf("No Channel Mentions"))
```

Comment:
`fmt.Sprintf` is redundant here -- no parameterization is occurring.
```suggestion
		mlog.Debug("No Channel Mentions")
```

However, I'd be inclined to just remove this debugging statement altogether -- I can see it being useful while iterating, but I'm not sure it's valuable in a production instance.